### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Accredited Representation Management: Representative Status in Profile and MyVA

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#88955]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Apologies for the slow reply -- turns out COVID is no fun. Thank you for your patience!
- This all looks good! Really minor feedback below.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Must:** I assume we talked about this when we saw the widget previously, but I don't 100% remember... when a link opens in a new window, make sure there's some kind of warning for screen reader users. Easiest option is on-screen visible text "(opens in a new window)" but including it as part of an `aria-label` is fine too. (Note that I'm not worried about the telephone links, which are a different experience overall and don't need that warning.)
- [x] **Consider:** Matt made a comment in the meeting about the ["How to replace" additional info beneath the card](https://www.figma.com/design/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?node-id=7528-79326&t=ZYkkOqRxDrTd5ZRZ-4), and whether that needs to be in an additional info. I think Matt's instincts here are correct --- unless there's a good reason to have it collapsed for most users, this might work better as just a heading plus text beneath the heading.
   
   The additional info component is fully accessible, but an H2 heading here will be marginally more discoverable for screen reader users. Likewise, one less click marginally removes friction.
   
   But I'm not going to lose sleep over this. I think it's fine as is, but may be worth exploring without the additional info.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by oddball-lindsay** [comment]
The address link will open in a new tab, and we have included an annotation for development to include the aria-label.  This will be consistent with how the Representative Status Widget is setup.

<img width="1098" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMFFUQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--11c6523d841a83c21a0b7f940dfd06f1f3cc61c0/image.png" alt="image.png" />
